.luacheckrc

```diff
@@ -12,8 +12,14 @@ stds.roblox = {
 	}
 }
 
-std = "lua51+roblox"
+stds.testez = {
+	read_globals = {
+		"it", "describe", "beforeAll", "beforeEach", "afterAll", "afterEach",
+	},
+}
+
+std = "lua51+roblox+testez"
```

Comment:
When we're testing TestEZ itself, we don't generally have access to TestEZ, so I'm not sure that adding it to our Luacheck config is correct here.

Reply:
If I dont add it here, Tests/lifecycleHooks.lua wont pass luacheck since we are testing these globals

Reply:
We should narrow down the scope of what files we inject these globals into, then. I don't want to make Luacheck think that the entire TestEZ codebase has access to these values, it mostly doesn't.

We've got two options I think:
1. Use Luacheck assertions on the functions that use these methods, if there's a small number of them.
2. Only apply the testez std for the tests folder or even the specific test that depends on these.

Reply:
2. seems to be the most applicable, since this sounds like a problem of scoping.
We can create a new luacheckrc file for the tests directory that has the testez std. Would that work?

Reply:
We don't need to make a new luacheckrc file, just scope the rules using globs to only affect the files in the tests folder like we do in Roact:

https://github.com/Roblox/roact/blob/master/.luacheckrc